masterlist.yaml

```diff
@@ -4173,6 +4179,12 @@ plugins:
         condition: 'active("Skyrim Revamped - Complete Enemy Overhaul.esp") and not active("PAN_NPCs_DG_SRCEO_Patch.esp")'
   - name: 'PAN_NPCs_DG_MLU_Patch.esp'
     after: [ 'MLU.esp' ]
+  - name: 'AI Overhaul - PAN_NPCs Patch.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19012/' ]
+    after: [ 'AI Overhaul - Cutting Room Floor Patch.esp' ]
+    clean:
+      - crc: 0x2611A250
+        util: 'SSEEdit v4.0.3'
```

Comment:
Generally I'll keep plugins from the same page together.
It's not a blocking issue though and can be commited as it is.

```yaml
  - name: 'PAN_NPCs.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19012/'
        name: 'Pandorable''s NPCs'
    msg:
      - <<: *patchProvided
        subs: [ 'AI Overhaul' ]
        condition: 'active("AI Overhaul.esp") and not active("AI Overhaul - PAN_NPCs Patch.esp")'
  - name: 'AI Overhaul - PAN_NPCs Patch.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19012/' ]
    after: [ 'AI Overhaul - Cutting Room Floor Patch.esp' ]
    clean:
      - crc: 0x2611A250
        util: 'SSEEdit v4.0.3'

  - name: 'PAN_NPCs_DB.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/30680' ]
    msg:
      - <<: *patchProvided
        subs: [ 'AI Overhaul' ]
        condition: 'active("AI Overhaul.esp") and not active("AI Overhaul - PAN_NPCs_DB Patch.esp")'
```

Reply:
my bad, i'll create a pull request to fix it